<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-GAD15EJN" name="wxp_ShoppingCart.TestToPayAddAddress" tests="13" failures="0" timestamp="8 Dec 2019 02:06:24 GMT" time="146.797" errors="1">
  <testcase name="test_AddAddressFail" time="11.138" classname="wxp_ShoppingCart.TestToPayAddAddress"/>
  <testcase name="test_AddAddressFail" time="10.982" classname="wxp_ShoppingCart.TestToPayAddAddress"/>
  <testcase name="testAddAddressPass" time="8.543" classname="wxp_ShoppingCart.TestToPayAddAddress"/>
  <testcase name="test_AddGood01" time="14.068" classname="wxp_ShoppingCart.TestToPayAddAddress"/>
  <testcase name="test_AddAddressFail" time="11.256" classname="wxp_ShoppingCart.TestToPayAddAddress"/>
  <testcase name="test_fix_ReceiptPerson" time="0.000" classname="wxp_ShoppingCart.TestToPayAddAddress">
    <error type="org.testng.TestNGException" message="
Method test_fix_ReceiptPerson requires 2 parameters but 0 were supplied in the @Test annotation.">
      <![CDATA[org.testng.TestNGException: 
Method test_fix_ReceiptPerson requires 2 parameters but 0 were supplied in the @Test annotation.
	at org.testng.internal.Parameters.checkParameterTypes(Parameters.java:191)
	at org.testng.internal.Parameters.createParameters(Parameters.java:127)
	at org.testng.internal.Parameters.createParameters(Parameters.java:376)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:451)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1276)
	at org.testng.internal.Invoker.createParameters(Invoker.java:991)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1081)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
    </error>
  </testcase> <!-- test_fix_ReceiptPerson -->
  <testcase name="test_AddAddressFail" time="11.111" classname="wxp_ShoppingCart.TestToPayAddAddress"/>
  <testcase name="test_ClickGoBackCart" time="4.111" classname="wxp_ShoppingCart.TestToPayAddAddress"/>
  <testcase name="test_fix_ReceiptCompany2" time="16.596" classname="wxp_ShoppingCart.TestToPayAddAddress"/>
  <testcase name="testClickPaytotal" time="4.176" classname="wxp_ShoppingCart.TestToPayAddAddress"/>
  <testcase name="testLogin" time="36.297" classname="wxp_ShoppingCart.TestToPayAddAddress"/>
  <testcase name="testPaytotal" time="8.331" classname="wxp_ShoppingCart.TestToPayAddAddress"/>
  <testcase name="test_fix_ReceiptCompany" time="10.188" classname="wxp_ShoppingCart.TestToPayAddAddress"/>
</testsuite> <!-- wxp_ShoppingCart.TestToPayAddAddress -->
